.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "editor.codeActionsOnSave": {
```

Comment:
Not sure that this works. If I add an easily fixed linter issue, nothing seems to happen after I save. Also potentially related, there is a warning related to this entry, when I open in Gitpod: "Property editor.codeActionsOnSave is not allowed"

Reply:
Seems to work locally if I use VS Code. Maybe that's good enough? Though ideally it would work in Theia-based applications, like Gitpod.

Reply:
It works in vscode and it's in the .vscode directory fwiw. Could it be a bug/missing feature in theia? Or is it expected that it should work?

Reply:
Yes, it should work I think. Theia apps will use the .vscode folder, if present.

Reply:
I noticed that the `eslint` VS Code extension is not part of the app. But even adding it is not enough. Maybe we can look at making linting work in Theia apps afterwards, if this was not originally wanted for this PR.